anndata/core/raw.py

```diff
@@ -16,20 +16,28 @@
 class Raw:
     def __init__(
         self,
-        adata: Optional['anndata.AnnData'] = None,
+        adata: 'anndata.AnnData',
         X: Union[np.ndarray, sparse.spmatrix, None] = None,
-        var: Union[AxisArrays, AxisArraysView, None] = None,
-        varm: Union[AxisArrays, AxisArraysView, None] = None,
+        var: Union[
+            AxisArrays, AxisArraysView, Mapping[str, Sequence], None
+        ] = None,
```

Comment:
`var: Optional[pd.DataFrame] = None`?

Reply:
Mapping also works right?

Reply:
I'd lean no, but do we ever do that? I think `pd.DataFrame` is probably most accurate.

Definitely not `AxisArrays` or `AxisArraysView` though.

Reply:
Yeah, it’s tested in `test_base.test_creation`: https://github.com/theislab/anndata/blob/b0e1a004e64d0e3aadbb9265e0a2fd7ead27b416/anndata/tests/test_base.py#L29-L35